tests/scripts/local/flake8_plugin_test.py

```diff
@@ -0,0 +1,87 @@
+import ast
```

Comment:
I adapted these tests to not have filesystem fixtures (more like how `flake8` will call this) -- which also avoids the need to self-exclude

otherwise the tests are the same